### ISSUE TITLE
Why can the loss function be simplified by clamp

### ISSUE THREAD

**Issue by jiangming0chen** [carlomarxdk/RobustCrossEntropyLoss#1]
The paper [Making deep neural networks robust to label noise: A loss correction approach] points out to estimate the noise transfer matrix T, and then retain the network. Why can the loss function be simplified by pred = torch.clamp(pred.softmax(-1), min = eps, max = 1-eps)?
thanks.
